ui/components/ui/multi-select.tsx

```
@@ -0,0 +1,1066 @@
+                </div>
+              ) : (
+                <div className="mx-auto flex w-full items-center justify-between">
+                  <span className="text-muted-foreground mx-3 text-sm">{placeholder}</span>
+                  <ChevronDown className="text-muted-foreground mx-2 h-4 cursor-pointer" />
+                </div>
+              )}
+            </Button>
+          </PopoverTrigger>
+          <PopoverContent
+            id={listboxId}
+            role="listbox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)